test_ffi/tests/ffi_callback_errors.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+const targetDir = Deno.execPath().replace(/[^\/\\]+$/, "");
+const [libPrefix, libSuffix] = {
+  darwin: ["lib", "dylib"],
+  linux: ["lib", "so"],
+  windows: ["", "dll"],
+}[Deno.build.os];
+const libPath = `${targetDir}/${libPrefix}test_ffi.${libSuffix}`;
+
+const dylib = Deno.dlopen(
+  libPath,
+  {
+    store_function_2: {
+      parameters: ["function"],
+      result: "void",
+    },
+    call_stored_function_2: {
+      parameters: ["u8"],
+      result: "void",
+    },
+    call_stored_function_2_from_other_thread: {
+      name: "call_stored_function_2",
+      parameters: ["u8"],
+      result: "void",
+      nonblocking: true,
+    },
+    call_stored_function_2_thread_safe: {
+      parameters: ["u8"],
+      result: "void",
+    },
+  } as const,
+);
+
+globalThis.addEventListener("error", (data) => {
+  console.log("Unhandled error");
+  data.preventDefault();
+});
+globalThis.onerror = (data) => {
+  console.log("Unhandled error");
+  if (typeof data !== "string") {
+    data.preventDefault();
+  }
+};
+
+globalThis.addEventListener("unhandledrejection", (data) => {
+  console.log("KEKKONEN");
```

Comment:
Oops. This isn't really proper logging even in a test :D Nothing dirty but nothing meaningful either.